### PR TITLE
Make logstash version configurable and default to 2.3.x. Fix an issue…

### PR DESCRIPTION
**Pull request by ecassady**
… with the 2.3 init script.

@dwcramer and @ericleeevans please review.

### CODE REVIEW

**Comment by dwcramer** [comment]
LGTM

File: files/init.d/logstash.patch [hunk 1]
```diff
@@ -0,0 +1,11 @@
+--- /etc/init.d/logstash.orig	2016-05-19 17:54:36.004217940 +0000
++++ /etc/init.d/logstash	2016-05-19 17:54:47.164340285 +0000
+@@ -97,7 +97,7 @@
```
Comment (ericleeevans) [inline files/init.d/logstash.patch:3]: Eh, this looks like a git changes file. Is it supposed to look like this?

File: files/init.d/logstash.patch [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+       sleep 1
+     done
+     if status ; then
+-      if [ "$KILL_ON_STOP_TIMEOUT" == 1 ] ; then
++      if [ "$KILL_ON_STOP_TIMEOUT" -eq 1 ] ; then
```
Comment (ericleeevans) [inline files/init.d/logstash.patch:8]: Ditto above.

Comment (ecassady) [inline files/init.d/logstash.patch:8]: Yes, this is a patch to be applied by the ansible patch module.

**Comment by ericleeevans** [comment]
+!

**Comment by ericleeevans** [comment]
Er, +1

**Pull request merged**